### ISSUE TITLE
can't read "g_numberOfAnnotationCol(/Validacao/teste-annotsv/AnnotSV-3.1/share/AnnotSV/Annotations_Human/AnyOverlap/CytoBand/GRCh38/cytoBand_GRCh38.formatted.sorted.bed)": no such element in array

### ISSUE THREAD

**Issue by desouzareis** [lgmgeo/AnnotSV#68]
When I tried to run the example bellow using AnnotSV 3.1

`$ANNOTSV/bin/AnnotSV -SVinputFile test.bed -outputFile ./test.annotated.tsv -svtBEDcol 4`

I got the following error message, even though the file cytoBand_GRCh38.formatted.sorted.bed is present.

```
can't read "g_numberOfAnnotationCol(/Validacao/teste-annotsv/AnnotSV-3.1/share/AnnotSV/Annotations_Human/AnyOverlap/CytoBand/GRCh38/cytoBand_GRCh38.formatted.sorted.bed)": no such element in array
    while executing
"lrepeat $g_numberOfAnnotationCol($formattedSortedUserBEDfile) """
    (procedure "userBEDannotation" line 17)
    invoked from within
"userBEDannotation $cytobandBEDfile $SVchrom $SVleft $SVright"
    (procedure "OrganizeAnnotation" line 763)
    invoked from within
"OrganizeAnnotation"
    (file "/Validacao/teste-annotsv/AnnotSV-3.1//bin/AnnotSV" line 282)
```

**Comment by lgmgeo** [comment]
Hi desouzareis,

Sorry, this bug is caused by a stupid glitch...
For the moment, you need to remove the last "/" in your setting of the ANNOTSV environnemental variable:
Replace: ```setenv ANNOTSV /path/to/install/AnnotSV/```
with:       ```setenv ANNOTSV /path/to/install/AnnotSV```
It should work... please confirm (I will add a bugfix soon).

Best regards,
Véronique

**Comment by desouzareis** [comment]
Hi Véronique,

Thank you very much!

Best regards

**Comment by lgmgeo** [comment]
I just added a bug fix (v3.1.1).
Thanks for the report.
